test/export/test_export.py

```diff
@@ -1207,14 +1207,14 @@ def forward(self, x):
             str(gm.code).strip(),
             """\
 def forward(self, arg_0):
-    l_x_, = fx_pytree.tree_flatten_spec(([arg_0], {}), self._in_spec)
+    arg7_1, = fx_pytree.tree_flatten_spec(([arg_0], {}), self._in_spec)
```

Comment:
I think for debugging purposes I'd like to keep the original node names like `l_x_`